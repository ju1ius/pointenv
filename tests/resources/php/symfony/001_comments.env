# FOO=bar # comment
EMPTY=  # comment
A=a # comment
B=not#a_comment
C='c' # comment
D='not'#a_comment
E="e" # comment
F="not"#a_comment
