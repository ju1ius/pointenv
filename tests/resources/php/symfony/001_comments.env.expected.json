{
    "EMPTY": "",
    "A": "a",
    "B": "not#a_comment",
    "C": "c",
    "D": "not#a_comment",
    "E": "e",
    "F": "not#a_comment"
}